netbox/utilities/exceptions.py

```diff
@@ -24,6 +24,14 @@ def __init__(self, message):
         self.message = message
 
 
+class AbortScript(Exception):
+    """
+    Raised to cleanly abort a script.
+    """
+    def __init__(self, message):
+        self.message = message
```

Comment:
Message assignment is actually handled for us by the base class, so we don't need to replicate it explicitly.
```suggestion
    pass
```